pkg/generators/openapi.go

```diff
@@ -242,6 +242,16 @@ func methodReturnsValue(mt *types.Type, pkg, name string) bool {
 	return r.Name.Name == name && r.Name.Package == pkg
 }
 
+func hasOpenAPIV3DefinitionMethod(t *types.Type) bool {
+	for mn, mt := range t.Methods {
+		if mn != "OpenAPIV3Definition" {
+			continue
+		}
+		return methodReturnsValue(mt, openAPICommonPackagePath, "OpenAPIDefinition")
```

Comment:
OpenAPIDefinition or OpenAPIV3Definition?

Reply:
it's the name of struct `OpenAPIDefinition` under "pkg/common" package, instead of the function name